DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,309 @@
+GLOBAL_LIST_INIT(EMH_blacklist, list())
+
+//EMITTER
+
+/obj/machinery/emh_emitter
+	name = "Emergency medical hologram emitter"
+	desc = "Simply click this device and an emergency medical hologram will be summoned. Swipe it with a high level ID to terminate the current EMH if it's misbehaving."
+	icon = 'DS13/icons/obj/decor/wall_decor.dmi'
+	icon_state = "emh-off"
+	anchored = TRUE
+	density = FALSE
+	pixel_y = 32 //Put on the tile below a wall etc etc modular fun time mapping mayhem
+	var/range = 10 //10 tiles? Pretty damn generous.
+	var/mob/living/carbon/human/species/holographic/emh
+	req_access = list(ACCESS_HEADS)
+	var/locked = FALSE //Cooldown to prevent spam
+
+/obj/machinery/emh_emitter/Initialize()
+	. = ..()
+	START_PROCESSING(SSmachines,src)
```

Comment:
Instead rework it so instead of needing to process all the time its only once a EMH moves to USE this emh_emitter it starts to use it.

Also I believe putting the get_area for first name of this is a VERY suitable thing to do.